### PR TITLE
homeworks/html-css/ivan_mytsak

### PR DESCRIPTION
**Pull request by DYFIZ**
actually it's just a some way to solve this task

### CODE REVIEW

**Comment by DYFIZ** [comment]
just a close it branch

File: homeworks/html-css/ivan_mytsak/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+</head>
+<body>
+	<!-- template for html-css task	-->
+<div class="ny_container">
+	<header>
+		<h1>new york,ny</h1>
+		<div class="t-date">Tuesday,April 15th <br />Overcast</div>
+		<div class="t-container">	
+			<div class="temperature">
+					<img src="../icons/png/iconfinder_weather04_4102315.png"/>
```
Comment (maxkharena) [inline homeworks/html-css/ivan_mytsak/index.html]: alt attr for img is required

File: homeworks/html-css/ivan_mytsak/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+				<div class="f-item f-below">Pollen 36</div>
+	</div>	
+	<div class="f-container">
+				<div class="f-item f-day">Sunday</div>
+				<div class="f-item f-img"><img src="../icons/png/iconfinder_weather05_4102316.png" alt="raining"></div>
+				<div class="f-item"><span>64 &#176;</span><br /><span class="f-below">50 &#176;</span></div>
+				<div class="f-item f-below">Pollen 36</div>
+	</div>	
+		<div class="f-container">
+				<div class="f-item f-day">Monday</div>
```
Comment (maxkharena) [inline homeworks/html-css/ivan_mytsak/index.html]: remove extra tab
1 tab after the block is required
actual 2 tabs

File: homeworks/html-css/ivan_mytsak/style.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+  margin: 0;
```
Comment (maxkharena) [inline homeworks/html-css/ivan_mytsak/style.css]: tab required

File: homeworks/html-css/ivan_mytsak/index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>	html-css task</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<!-- template for html-css task	-->
+<div class="ny_container">
```
Comment (maxkharena) [inline homeworks/html-css/ivan_mytsak/index.html]: You can just add flex to body tag without using` <div class="ny_container">`

**Pull request merged**